edx_sga/static/css/edx_sga.css

```diff
@@ -140,9 +143,15 @@
 }
 
 .sga-block a.button {
-    background-color: #E5E4E3;
-    border: 1px solid #cec5c5;
-    border-radius: 2px;
+    box-shadow: inset 0 1px 0 0 #fff;
+    background-image: -webkit-linear-gradient(#f1f1f1,#d9d1d1);
```

Comment:
please refactor these classes and pull common data in new class , things are repeating.
i.e
  background-image: -webkit-linear-gradient(#f1f1f1,#d9d1d1);
 +    background-image: linear-gradient(#f1f1f1,#d9d1d1);
 +    text-shadow: 0 1px 0 #fcfbfb;
 +    background-clip: padding-box;